js/components/directionControlVr.jsx

```diff
@@ -19,7 +21,15 @@ class DirectionControlVr extends React.Component {
    * @param {Event} event - event object
    */
   rotateVrVideo(event) {
-    const isRotated = event && (event.type === 'mousedown' || event.type === 'touchstart');
+    const charCode = event.which || event.keyCode;
```

Comment:
is it correct to check `event.which || event.keyCode` on `onMouseDown` or `onTouchStart` events?

Reply:
2 functions for these types of events were added(as you wanted :-) )